lib/client.coffee

```diff
@@ -9,29 +20,74 @@ module.exports = class IntuitClient
     [body, done] = [{}, body] if typeof(body) is "function"
     new Request(@options)[method]("#{BASE_URL}#{path}", body, done)
 
-  institutions: (done) ->
-    @request "get", "/institutions", done
+  institutions: (userId, done) ->
+    @options.userId = userId
```

Comment:
Individual userId. Maybe there's a better way to accomplish this, but essentially I want the flexibility of passing in any userId here which is just referenced from the global instance options when signing the SAML request to get the OAuth token. This is the only option that is used in the signature that isn't global